### ISSUE TITLE
Last used webspace captures user if webspace does not exist anymore

### ISSUE THREAD

**Issue by popoplanter** [sulu/sulu#7440]
| Q | A
| --- | ---
| Sulu Version | 2.6.2
| PHP Version | -
| DB Version | -

# Actual Behavior

When navigating to /admin/#/webspaces/plscaptureme and then navigating to the webspaces tab in the side navigation of the admin area, sulu wants to load a not existing webspace "plscaptureme" and the only way out is to manually enter an existing webspace with /admin/#/webspaces/webspace AND then clicking on webspaces in the navigation immediatelly after the site loads OR change the last used webspace key in the se_user_settings table. This captures the user in an infinite loop of trying to get to the pages but getting redirected to a non existing webspace. Can occur when deleting a webspace or manually typing a wrong webspace value in the url.

# Expected Behavior

If a webspace does not exist, default to the default webspace.

# Steps to Reproduce
go to /admin/#/webspaces/plscaptureme and click on webspaces in the navigation.

**Comment by alexander-schranz** [comment]
I think this should be fixed in https://github.com/sulu/sulu/pull/7378 by @mamazu please check if you are using the latest version of Sulu.

Please provide always the version or versions which you did test the behaviour not just write All.

**Comment by mamazu** [comment]
Nope, this is a javascript problem. The `WebspaceStore` tries to find a webspace (in the `getWebspace` function) that doesn't exist in the list of available webspaces:
https://github.com/sulu/sulu/blob/a57671127bde0227699c4e609052a514ecba9c98/src/Sulu/Bundle/PageBundle/Resources/js/stores/webspaceStore/webspaceStore.js#L27

My PR only fixes the behaviour of you select a webspace (which exists) and you don't have permission.
